flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/operations/AlterSchemaConverter.java

```diff
@@ -92,7 +92,7 @@ public class AlterSchemaConverter {
     private final Function<SqlNode, String> escapeExpression;
     private final CatalogManager catalogManager;
 
-    AlterSchemaConverter(
+    public AlterSchemaConverter(
```

Comment:
We add `public` to reuse `AlterSchema` common class. When all `SqlAlterTables` migrate to `SqlNodeConverter` we can move AlterSchema under converters package and reset `public` identifier.